Adds debian bullseye support

gurkanindibay opened the pull request:
Adds debian bullseye among supported os list

hanefi commented:
What about the `deb.sh` files in other directories? Should we remove them completely in a separate PR?

gurkanindibay commented:
> What about the `deb.sh` files in other directories? Should we remove them completely in a separate PR?

I changed them all please check files section I changed 4 files
